crates/bevy_render/src/camera/camera.rs

```diff
@@ -61,7 +61,7 @@ impl Default for Viewport {
     fn default() -> Self {
         Self {
             physical_position: Default::default(),
-            physical_size: Default::default(),
+            physical_size: [1, 1].into(),
```

Comment:
```suggestion
            physical_size: UVec2::new(1, 1),
```

(Nit) Could you use `UVec2::new` instead? I feel like its a bit clearer, and because it is a `const fn` it hints to the compiler than it can be inlined.

Reply:
> because it is a const fn it hints to the compiler than it can be inlined.

`const fn` doesn't do that as far as I know. The `Into`/`From` impls involved are trivial enough that the compiler can figure this out on its own.

Reply:
Ok, I won't push it. :)

Reply:
I've made the change, I won't argue that it's clearer (though IMO glam doesn't have a "clear" API no matter which part you use anyways)